apps/ledger-live-desktop/src/renderer/families/tezos/DelegateFlowModal/steps/StepSummary.tsx

```diff
@@ -162,12 +161,6 @@ const StepSummary = ({
                   <Placeholder />
                 )}
               </Container>
-
-              {isRandomChoice ? (
-                <Text ff="Inter|Medium" color="palette.text.shade60" fontSize={2}>
-                  <Trans i18nKey="delegation.flow.steps.summary.randomly" />
```

Comment:
We should remove the key if it isn't used anywhere else, i didn't check

Reply:
It's used